### PR TITLE
Added ProSampler recipe

### PR DESCRIPTION
**Pull request by simonvh**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/prosampler/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+$GXX -o ProSampler ProSampler.cc
```
Comment (dpryan79) [inline recipes/prosampler/build.sh]: Please change `$GXX` to `$CXX`.

File: recipes/prosampler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+package:
+  name: prosampler
+  version: {{ version }}
+
+source:
+  url: https://github.com/zhengchangsulab/ProSampler/raw/master/PROSAMPLER_01_22_2019.tar.gz 
+  sha256: {{ sha256 }}
+
+build:
+  skip: True # [osx]
```
Comment (dpryan79) [inline recipes/prosampler/meta.yaml]: This will likely build on OSX with a small compiler tweak.

File: recipes/prosampler/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('gxx') }}
```
Comment (dpryan79) [inline recipes/prosampler/meta.yaml]: Please change `gxx` to `cxx`.

**Comment by simonvh** [comment]
Thanks @dpryan79 , it now also works for osx!

**Comment by dpryan79** [comment]
Great! Merge at your leisure.

**Pull request merged**